Examples/php/value/runme.php

```diff
@@ -1,17 +1,15 @@
 <?php
 
-	require "example.php";
 
+	$v = new Vector();
+        $v->x_set(1.0);
+        $v->y_set(2.0);
+        $v->z_set(3.0);
```

Comment:
This example also currently builds with `-noproxy`, but aside from that I think this is good.